src/test/emmetHelper.test.ts

```diff
@@ -333,8 +338,8 @@ describe('Test custom snippets', () => {
 			return updateExtensionsPath(extensionsPath).then(() => {
 				let foundCustomSnippet = false;
 				const expandOptionsWithCustomSnippets = getExpandOptions('css');
-				expandOptionsWithCustomSnippets.snippets.all({ type: 'string' }).forEach(snippet => {
```

Comment:
Changing these tests cause the api they relied on have changed. Still this would be a breaking change for `vscode-emmet-helper` unless they also rely on this implementation detail.